### PR TITLE
Various style guide improvements

### LINKED ISSUE #3771: Style guide - changes part three
Bring in some more changes from the to-do list in our style guide questions doc

### PR DESCRIPTION
**Pull request by andrew-walford-prisma**
- Added "wish" to "do not use" words list
- Improved the front matter descriptions## Describe this PR
- Fixed heading level in writing style doc
- Added a new code examples section (with the requirement that we use constants in query examples)
- Linked to Prisma's internal terminology page

### CODE REVIEW

File: content/600-about/200-prisma-docs/20-style-guide/02-word-choice.mdx [hunk 1]
```diff
@@ -145,6 +145,7 @@ Avoid the following common words in English, because they are ambiguous to many
 | May (to mean "might")            | Can also mean "is permitted to"                    | "Might"                                                                              |
 | Should (to mean "ought to")      | Can also mean "might happen"                       | "Must": "You must rebuild Prisma Client", or <br></br>Omit: "Rebuild Prisma Client." |
 | Once (to mean "when" or "after") | Can also mean "happens once"                       | "When": "When the build completes..."                                                |
+| Wish (to mean "want"")           | Not always clear internationally                   | "Want"                                                                               |
```
Comment (janpio) [inline content/600-about/200-prisma-docs/20-style-guide/02-word-choice.mdx]:
```suggestion
| Wish (to mean "want")           | Not always clear internationally                   | "Want"                                                                               |
```

File: content/600-about/200-prisma-docs/30-docs-components/03-frontmatter.mdx [hunk 2]
```diff
@@ -1,34 +1,53 @@
...
 
 ## <inlinecode>title</inlinecode>
 
-The `<h1>` title of the page.
+The `<h1>` title of the page. This title appears at the top of the page, and in the left-hand navigation.
+
+Note:
+
+- You can wrap the `title` text in single quotes, but this is not mandatory.
+- If you wrap the `title` text in single quotes, then you cannot include apostrophes in the text.
```
Comment (janpio) [inline content/600-about/200-prisma-docs/30-docs-components/03-frontmatter.mdx:15]: You should be able to, you just need to escape it probably with a backslash?

Comment (andrew-walford-prisma) [reply content/600-about/200-prisma-docs/30-docs-components/03-frontmatter.mdx:15]: I did experiment with that a while back, at least in some of the other metadata fields, and I couldn't find a way to get it working. I even tried substitute characters like `&apos;`

File: content/600-about/200-prisma-docs/20-style-guide/02-word-choice.mdx [hunk 3]
```diff
@@ -145,6 +145,7 @@ Avoid the following common words in English, because they are ambiguous to many
 | May (to mean "might")            | Can also mean "is permitted to"                    | "Might"                                                                              |
 | Should (to mean "ought to")      | Can also mean "might happen"                       | "Must": "You must rebuild Prisma Client", or <br></br>Omit: "Rebuild Prisma Client." |
 | Once (to mean "when" or "after") | Can also mean "happens once"                       | "When": "When the build completes..."                                                |
+| Wish (to mean "want")           | Not always clear internationally                   | "Want"                                                                               |
```
Comment (andrew-walford-prisma) [inline content/600-about/200-prisma-docs/20-style-guide/02-word-choice.mdx]:
```suggestion
| Wish (to mean "want")            | Not always clear internationally                   | "Want"                                                                               |
```

File: content/600-about/200-prisma-docs/30-docs-components/03-frontmatter.mdx (hunk 2 above)
Comment (andrew-walford-prisma) [reply content/600-about/200-prisma-docs/30-docs-components/03-frontmatter.mdx:15]: I just double-checked, and backslash doesn't do this. There might be a way to do it that I haven't found, but the workaround is to not use single quotes, which I shop the addition to the doc will remind us to do.

**Pull request merged**